src/main/java/com/club_board/club_board_server/domain/user/Department.java

```diff
@@ -37,6 +36,12 @@ public String getDisplayName() {
         return displayName;
     }
 
+    public static List<String> showDepartment() {
+        return Arrays.stream(Department.values())
+                .map(Department::getDisplayName)
+                .toList();
+    }
```

Comment:
```suggestion
    private static final List<String> DEPARTMENT_NAMES = Arrays.stream(Department.values())
            .map(Department::getDisplayName)
            .toList();

    public static List<String> showDepartment() {
        return DEPARTMENT_NAMES;
    }
```

현재 코드는 딱히 메모리에 올리지 않고 있긴 해서 캐싱을 하고 싶다면 이런 느낌의 코드로 바꿔주면 될 것 같습니당 근데 이거는 나중에 실제 서비스 했을 때 자주 호출되지 않는다면, 이대로도 깔끔하고 좋아서 바꾸지 않아도 괜찮을 것 같아요!

Reply:
반영하겠습니다!